{{cookiecutter.repo_name}}/Makefile

```diff
@@ -43,7 +52,26 @@ quality:
 	pep8 --config=.pep8 {{cookiecutter.repo_name}} *.py
 	pylint --rcfile=pylintrc {{cookiecutter.repo_name}} *.py
 
-validate: test quality
+serve:
```

Comment:
To run the created django app. Since we ask for port while initializing cookie cutter on which the app should run so thats why I added it here.